app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -613,3 +616,36 @@ private fun RunHikeBottomSheet(
         }
       }) {}
 }
+
+@Composable
+private fun ElevationGraphOnPercentage(
```

Comment:
Warning: Is it necessary to have a function for this? The ElevationGraph already handles cases when the completionpercentage is null, there is no need to catch that in this scope, or?

Reply:
Yes because of Sonar complaining about the complexity.

Reply:
If you use an elvis operator, this should work, since ElevationGrapghPercentage has a null check built in

Reply:
Yes you are right. Fixed